packages/menus/src/containers/MenuContainer.spec.js

```diff
@@ -254,17 +254,41 @@ describe('MenuContainer', () => {
     });
   });
 
-  describe('componentWillUnmount', () => {
-    it('removes mousedown event listener', () => {
-      const removeEventListenerSpy = jest.fn();
-
-      document.removeEventListener = removeEventListenerSpy;
+  describe('when the menu is open', () => {
+    beforeEach(() => {
       wrapper = mountWithTheme(basicExample({ onChange: onChangeSpy }), {
         enzymeOptions: { attachTo: document.body }
       });
-      wrapper.unmount();
 
-      expect(removeEventListenerSpy).toHaveBeenCalled();
+      findTrigger(wrapper).simulate('click');
+    });
+
+    describe('when clicking outside', () => {
+      it('closes the menu', () => {
+        wrapper.simulate('click');
+
+        expect(findMenu(wrapper)).not.toExist();
+      });
+
+      it('removes click outside event listener', () => {
+        const removeEventListenerSpy = jest.fn();
+
+        document.removeEventListener = removeEventListenerSpy;
```

Comment:
Should we restore the original method not to temper with future tests?
Not sure what's the cleanest way do this with `jest` but with `sinon` it looks something like this:
``` js
const removeEventListenerSpy =  sinon.spy(document, 'removeEventListener');
// ... assert
removeEventListenerSpy.restore();
```

Reply:
It is a good point, I just copied it from the old test. I'll just fix that.

Reply:
The update looks good 👍